tests/errors/semantic/blocking/ADD_SET_UNHASHABLE_TYPE_1.py

```diff
@@ -0,0 +1,6 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+a = {1,2,3}
+b = [4,5]
+a.add(b)
```

Comment:
Before worrying about hashability, this is a problem of incompatible types! The set contains integers.

Reply:
The idea is to attempt adding a hashable type to test for errors in the pop method.

Reply:
Sorry, I cannot understand what you mean. What has the "pop" method to do with this PR?

Reply:
> Sorry, I cannot understand what you mean. What has the "pop" method to do with this PR?

sorry, I was  talking about the add Method.

Reply:
In any case, the test is for the constructor, not for the `add` method. I explained in another thread.